src/wireviz/Harness.py

```diff
@@ -27,7 +29,8 @@ def loop(self, connector_name, from_pin, to_pin):
 
     def connect(self, from_name, from_pin, via_name, via_pin, to_name, to_pin):
 
-        for (name, pin) in zip([from_name, to_name], [from_pin, to_pin]): # check from and to connectors
+        # check from and to connectors
+        for (name, pin) in zip([from_name, to_name], [from_pin, to_pin]):
```

Comment:
I'm happy to merge this PR, but please undo this change; I'm a big fan of short and sweet comments tucked away at the end of a line, since it doesn't break the flow while reading the code.

Feel free to amend and force-push your commit to keep the history clean. Thanks!

Reply:
Resolved.  Thanks, j